### ISSUE TITLE
make a GitHub org and keep all our (2.12) forks there

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#322]
at https://github.com/scala/community-builds/pull/320 @adriaanm writes:

> It was kind of nice to have a top-level overview of which community projects we had to fork, but maybe we can actually do even better in the new format by having a naming convention for the branches that constitute these forks, or by creating a github user that has the forked repos for the community build, so we have a centralized spot to track our forks.

acting as multiple GitHub users is too annoying IMO, but a new org would work

**Comment by SethTisue** [comment]
before making the new org, we should decrease the number of unnecessary forks.

done:
- [x] migration-manager
  - branch pushed directly to origin
- [x] specs2
  - unforking PR: https://github.com/scala/community-builds/pull/333
- [x] breeze
  - unforking PR: https://github.com/scala/community-builds/pull/334
- [x] ssl-config
  - upstream PR: https://github.com/typesafehub/ssl-config/pull/48
- [x] argonaut
  - upstream PR: https://github.com/argonaut-io/argonaut/pull/247
- [x] macro-paradise
  - https://github.com/scalamacros/paradise/issues/86
- [x] spire
  - unforking PR: https://github.com/scala/community-builds/pull/346
- [x] scalariform
  - upstream PR: https://github.com/scala-ide/scalariform/pull/233
- [x] parboiled
  - upstream PR: https://github.com/sirthias/parboiled/pull/100
- [x] cats
  - waiting on https://github.com/typelevel/cats/issues/1377
- [x] scalatest
  - unforking PR: https://github.com/scala/community-builds/pull/337
- [x] scalatags
  - upstream PR: https://github.com/lihaoyi/scalatags/pull/142
- [x] jackson-module-scala
  - upstream PR: https://github.com/FasterXML/jackson-module-scala/pull/290
- [x] lift-json
  - upstream PR: https://github.com/lift/framework/pull/1812
- [x] genjavadoc
  - upstream PR: https://github.com/typesafehub/genjavadoc/pull/95

**Comment by SethTisue** [comment]
2.11 still has tons of forks, but I think we should shrug and say so what, the action's on 2.12

**Comment by dwijnand** [comment]
sbinary is moribund in terms of sbt 1.0

**Comment by SethTisue** [comment]
lately we froze some projects because they added wartremover which confuses dbuild. it's #364

**Comment by milessabin** [comment]
Given that wartremover is a linter, wouldn't it be better to patch the projects to disable it rather than freeze them? Also, are the upstream projects aware of the issue with wartremover and the freezing?

**Comment by SethTisue** [comment]
the GitHub org now exists: https://github.com/scalacommunitybuild

I will migrate forks there, time permitting. (I don't have time right now to tackle it all at once, but at minimum we can make our new forks there.)

**Comment by SethTisue** [comment]
all the forks are now on the new org

to do before closing this ticket:

* [x] document the new org
* [x] do something with the forks list above? (make a new ticket?)
